libs/designsystem/src/lib/components/page/page.module.ts

```diff
@@ -1,19 +1,20 @@
-import { NgModule } from '@angular/core';
 import { CommonModule } from '@angular/common';
+import { NgModule } from '@angular/core';
 import { IonicModule } from '@ionic/angular';
 
 import { FitHeadingModule } from '../../directives/fit-heading/fit-heading.module';
+
+import { PageFooterComponent } from './page-footer/page-footer.component';
 import {
-  PageComponent,
   PageActionsComponent,
   PageActionsDirective,
+  PageComponent,
```

Comment:
Please revert unrelated file

Reply:
It looks like the pre commit script is doing this, so even if I make the corrections, and try to stage it, pre commit script just reverts it, giving me a commit with no changed files...

Reply:
See for example [9087f90](https://github.com/kirbydesign/designsystem/pull/1228/commits/9087f9006574b429f533230790ee6400b18383b6)

Reply:
Solved by merging master into the branch again. Not sure anymore what caused this.